### PR TITLE
Added favState to model and test

### PR DESCRIPTION
**Pull request by gewenyu99**

### CODE REVIEW

File: lib/models/request.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ const RequestSchema = new Schema({
 	created_at: { type: Date, default: Date.now },
 	created_by: { type: String, required: true },
 	status: { type: String, enum: Object.values(STATUSES) },
+  favState: { type: Boolean, required: true },
```
Comment (Jhong098) [inline lib/models/request.ts]: Can you fix the indentation?

Comment (gewenyu99) [reply lib/models/request.ts]: oops got it

File: docs/swagger.yml [hunk 2]
```diff
@@ -128,6 +129,8 @@ definitions:
         format: date-time
       created_by:
         type: string
+      fav_state:
+        type: bool
```
Comment (ilPikachu) [inline docs/swagger.yml]: boolean

Comment (gewenyu99) [reply docs/swagger.yml]: I'll get to this after work

**Pull request merged**